examples/wifi-echo/server/esp32/main/gen/clusters-callback-stubs.c

```diff
@@ -722,20 +745,20 @@ bool emberAfIasZoneClusterZoneStatusChangeNotificationCallback(uint16_t zoneStat
  *
  * @param identifyTime   Ver.: always
  */
-bool emberAfIdentifyClusterIdentifyCallback(uint16_t identifyTime)
-{
-    return false;
-}
+// bool emberAfIdentifyClusterIdentifyCallback(uint16_t identifyTime)
```

Comment:
please just delete these.  they were placed in this file by me to enable a "plugin-oriented" gen/ directory to be able to link with no plugins in the build

Reply:
in fact this whole file should end up empty and should be deleted after we've finished the port